data/human/news.txt

```diff
@@ -1463,6 +1479,25 @@ news "tourist"
 		word
 			`"`
 
+news "tourist [frozen]"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Independent" "Neutral"
+		attributes "frozen"
+	name
+		word
+			"Adventure tourist"
+			"Arctic tourist"
+			"Tourist"
+	message
+		word
+			`"`
+		word
+			"Have you ever seen so much white?"
+			"This place has such a stark beauty."
+			"I know this isn't the usual sort of tourist destination, but I like getting off the beaten path."
+			"Enjoying a hot tub while in the bitter cold really is the sort of pleasure that just isn't available in the Paradise Worlds."
```

Comment:
Worth noting that there *is* a paradise peripheral world famed for its winter sports and resorts. Chances are they probably have hot tubs there...